src/pages/settings/panes/Audio.tsx

```diff
@@ -27,11 +27,11 @@ export function Component() {
 
     const askOrGetPermission = async () => {
         try {
-            const mediaStream = await navigator.mediaDevices.getUserMedia(
+            const result = await navigator.mediaDevices.getUserMedia(
```

Comment:
the name `mediaStream` is already used in upperscope which result in strange behaviors in debugger